modules/rtgtools/vcfeval/main.nf

```diff
@@ -0,0 +1,48 @@
+process RTGTOOLS_VCFEVAL {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::rtg-tools=3.12.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/rtg-tools:3.12.1--hdfd78af_0':
+        'quay.io/biocontainers/rtg-tools:3.12.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(query_vcf), path(query_vcf_tbi), path(bed)
```

Comment:
Is the bed file related to the query or the reference vcf? Or is it just a set of truth regions?
If so, I think it would be better to decouple it from the sample tuple

Reply:
I think it just needs to consist of some truth regions, but maybe I should just decouple them indeed

Reply:
Would it make sense to rename the variable to something more explicit then? e.g. `truth_regions` instead of `bed`?

Reply:
Good suggestion! I'll change that